update laravel and guzzle requirements

pincombe opened the pull request:
Bumped Guzzle and Laravel requirements to support Laravel 8

pincombe committed:
update laravel and guzzle requirements

pincombe committed:
Added Laravel 8 Keyword

pincombe committed:
Bumped version

mostafaznv commented on composer.json:
```diff
@@ -28,8 +28,8 @@
-        "laravel/framework": "5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || 6.* || 7.*",
-        "guzzlehttp/guzzle": "6.*"
+        "laravel/framework": "5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || 6.* || 7.* || 8.*",
+        "guzzlehttp/guzzle": "7.*"
```
I think it should be "6.* || 7.*" to support older versions of laravel and guzzle

pincombe replied:
@mostafaznv you're right. I've updated the pull request to reflect the change 👍

pincombe committed:
added support for guzzle 6 

Added support for guzzle 6 to allow older versions of laravel to function still
